### DISCUSSION TITLE
print styles with Tailwind not working

### DISCUSSION THREAD

**Discussion by ali1059** [tailwindlabs/tailwindcss#13399] [Help]
I have added print style in tailwind.config.css and the using print:hidden in class  but it does not remove components on print preview

  theme: {
    extend: {
      screens: {
        print: { raw: 'print' },
        screen: { raw: 'screen' },
      },
      colors: colors,
    },
  },

**Comment by mutasim77** [comment]
`print` is built-in after tailwind v3. Just use it without any configs:
```html
<article class="print:hidden">
   Don't Print Me ;)
</article>
```

**Reply by ali1059** [reply]
@mutasim77 Thanks for responding, the issue is print:hidden is not working for me on the dev server, but its working on localhost on rails vue app,that's why I added it in the config
